### PR TITLE
Publish the style guide to the Ballerina website

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is to add the content and publish the Ballerina Style Guide to the Ballerina Website.

Fixes https://github.com/ballerina-platform/ballerina-www/issues/1449

- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/learn/style-guide/README.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
+# Ballerina Style Guide
```
Comment (NipunaMarcus) [inline website/two-column-pages/docs/learn/style-guide/README.md]: We don't need this file as you already have added the same content into the learn/style-guide.md

File: website/two-column-pages/docs/learn/style-guide/definitions.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+    }
+    ```
+
+* Block indent each function definition, resource definition, and field definition inside a service definition.
+ 
+## Object definition
+
+* Block indent each field definition and each function definition on their own line.
+* Init function should be placed before all the other functions. 
+* For function definitions in the object definition, follow the [function formatting guidelines](#function-definition).
```
Comment (NipunaMarcus) [inline website/two-column-pages/docs/learn/style-guide/definitions.md]: did we check whether link like this works ... ?

File: website/two-column-pages/docs/learn/style-guide/expressions.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+  
+  **Don'ts**
+  ```ballerina
+  <
+      string
+  >
+  ```
+## Table literal
+
+* Follow [record literals](#record-literals) formatting when formatting a table block.
```
Comment (NipunaMarcus) [inline website/two-column-pages/docs/learn/style-guide/expressions.md]: did you check whether this link works?

File: website/two-column-pages/docs/learn/style-guide/expressions.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+  
+  //Or
+  
+  Person p = {
+      name: "john",
+      age: 20};
+  ```
+## Map literal
+
+* For Map literals, follow the same formatting guidelines as [record literals](#record-literals).
```
Comment (NipunaMarcus) [inline website/two-column-pages/docs/learn/style-guide/expressions.md]: did you check whether this style of links work?

File: website/two-column-pages/docs/learn/style-guide/definitions.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+* Keep the listener inline with the service signature.
+  
+  Example,
+  ```ballerina
+  service hello on new http:Listener(9090) {
+      ...
+  }
+  ```
+* When formatting resource functions and function definitions, block indent each element and
+  follow the [function formatting guidelines.](#function-definition).
```
Comment (NipunaMarcus) [inline website/two-column-pages/docs/learn/style-guide/definitions.md]: check whether this link work?

File: website/two-column-pages/docs/learn/style-guide/README.md (hunk 1 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/style-guide/README.md]: Deleted it.

File: website/two-column-pages/docs/learn/style-guide/definitions.md (hunk 2 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/style-guide/definitions.md]: Checked and fixed.

File: website/two-column-pages/docs/learn/style-guide/expressions.md (hunk 4 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/style-guide/expressions.md]: Checked and fixed.

File: website/two-column-pages/docs/learn/style-guide/definitions.md (hunk 5 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/style-guide/definitions.md]: Checked and fixed.

File: website/two-column-pages/docs/learn/style-guide/expressions.md (hunk 3 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/style-guide/expressions.md]: Checked and fixed.

**Pull request merged**